lib/Target/Verilog/VerilogEmitter.cpp

```diff
@@ -403,6 +405,17 @@ LogicalResult VerilogEmitter::printOperation(mlir::arith::ConstantOp op) {
   return success();
 }
 
+LogicalResult VerilogEmitter::printOperation(
+    mlir::UnrealizedConversionCastOp op) {
+  // e.g. assign x0 = $signed(v100);
```

Comment:
Is it always unsigned to signed?

Reply:
So far, I've placed a limit on unsigned to signed conversions only, but it looks like copybara hasn't pulled that change in yet...